website/src/pages/use-cases/code-reuse.tsx

```diff
@@ -0,0 +1,308 @@
+                        <p>
+                            Current tools don’t fully enable teams to innersource and use existing code to develop a
+                            more secure and coherent codebase.
+                            <span className="font-weight-bold"> What does that mean for you and your team?</span>
+                        </p>
+                        <ul>
+                            <li>Finding idiomatic code examples is challenging if you don't know where to look.</li>
+                            <li>
+                                Without easy access to existing patterns, developers default to implementing from
+                                scratch, unintentionally increasing complexity and confusion in the codebase.
+                            </li>
+                            <li>
+                                Engineering leaders lack the visibility needed to ensure teams adopt recommended
+                                packages and retire deprecated ones.
+                            </li>
+                        </ul>
+                    </div>
+                    <div className="col-lg-5 mt-8 text-center">
+                        <section className="case-studies__quote--in-content--section font-weight-normal">
+                            <h4 className="mb-3">FactSet ensures consistency across its entire codebase</h4>
+                            <p>
+                                “If I’m developing code for a library that might draw charts, for example, we don’t want
+                                30 different ways to draw a chart at FactSet. With Sourcegraph, I can search the code to
+                                find other chart examples, and simply copy the code. This saves us time and ensures
+                                consistency.”
+                            </p>
+                            <figcaption className="pt-2 text-muted text-center">
+                                &mdash; Joseph Majesky, Software Engineer at FactSet
+                            </figcaption>
+                        </section>
+                        <img className="my-4" src="/external-logos/factset-logo.svg" width="150" alt="Factset logo" />
+                        <Link
+                            className="d-block font-weight-bold"
+                            to="/case-studies/factset-migrates-from-perforce-to-github"
+                        >
+                            Read the case study
+                            <ArrowRightIcon className="ml-2" />
+                        </Link>
+                    </div>
+                </div>
+            </ContentSection>
+        </div>
+
+        <ContentSection>
+            <div className="row mt-lg-0 mt-5 justify-content-center">
+                <div className="d-flex flex-column mt-lg-6 mt-4 w-100 mx-3">
+                    <h1 className="font-weight-bold text-lg-center text-left mb-lg-6 mb-md-6">How Sourcegraph helps</h1>
```

Comment:
I think we should use `text-md-center` so it matches the `Get started with Sourcegraph` section

Reply:
I agree! This brings up adjusting `CustomCarousel` to center on `md` screens as well. Do you think that looks okay? If so, I'll make sure it plays well wherever else `CustomCarousel` is used too.
![Screen Shot 2022-04-27 at 5 23 55 PM](https://user-images.githubusercontent.com/59381432/165647504-06de5b56-ea63-456d-852d-d8ed2cd6fea3.png)

Reply:
Yes that would be great! Thanks Becca!